tests/e2e/tests/e2e/GitSsh.spec.ts

```diff
@@ -72,19 +75,24 @@ suite('Git with ssh workflow', async () => {
         await editor.waitText('Untitled-0', 'ssh-rsa');
     });
 
-
     test('Add a SSH key to GitHub side and clone by ssh link', async () => {
         const sshName: string = WorkspaceNameHandler.generateWorkspaceName('test-SSH-', 5);
         const publicSshKey = await cheGitAPI.getPublicSSHKey();
         await gitHubUtils.addPublicSshKeyToUserAccount(TestConstants.TS_GITHUB_TEST_REPO_ACCESS_TOKEN, sshName, publicSshKey);
         await cloneTestRepo();
-
+        // workaround for issue: https://github.com/eclipse/che/issues/19101
+        await ide.waitNotificationAndClickOnButton('Would you like to open the cloned repository?', 'Open');
+        await ide.waitIde();
     });
 
     test('Change commit and push', async function changeCommitAndPushFunc() {
         const currentDate: string = Date.now().toString();
-        await projectTree.expandPathAndOpenFile('Spoon-Knife', committedFile);
+        await projectTree.clickOnItem('Spoon-Knife/' + committedFile);
+        await editor.waitEditorOpened(committedFile);
+        await editor.waitTab(committedFile);
+        await editor.type(committedFile, 'D' + '\n', 1);
```

Comment:
minor: I think it can be simplified like:
```
await editor.type(committedFile, 'D\n', 1);
```
but up to you - maybe your case is more readable...

Reply:
Left as is, :)